README.rst

```diff
@@ -81,6 +81,15 @@ Configuration
 
 See this default config_ that you can edit for your own purposes.
 
+Configure access to Google Ad Manager
+-------------------------------------
+
+In order to use line-item-manager, you need to provide JSON private key file and configure access to your Google Ad manager account:
+
+1. In Google API Console generate private key file for service account
+2. In Google Ad Manager enable API access and create new services user with Administrator role.
+
+See _`detailed instructions <https://developers.google.com/ad-manager/api/authentication#oauth>`_ in documentation of GAM authentication.
```

Comment:
```suggestion
See `detailed instructions <https://developers.google.com/ad-manager/api/authentication#oauth>`_ in documentation of GAM authentication.
```

see: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages